grisera_api/participant/participant_service.py

```diff
@@ -0,0 +1,28 @@
+import requests
+from graph_api_config import graph_api_address
+from participant.participant_model import ParticipantIn, ParticipantOut
+
+
+class ParticipantService:
+    """
+    Object to handle logic of participants requests
+    """
+
+    def save_participant(self, participant: ParticipantIn):
+        """
+        Send request to graph api to create new participant
+
+        Args:
+            participant (ParticipantIn): Participant to be added
+
+        Returns:
+            Result of request as participant object
+        """
+        commit_body = {"labels": ["Participant"]}
+        response = requests.post(url=graph_api_address+"/nodes",
```

Comment:
Did you think about extracitng the `request.post` call to some generic "`graph_api_service`" (analogous to `database_service` in **graph_api**)? I know I've written about YAGNI, but this will be used several times in the near future :smile:

Reply:
Yes, I was planning it. Right now, there would be small use of it so I left it for later. Although it's not problem to do it now.